File: 2704-maximum-difference-by-remapping-a-digit/2704-maximum-difference-by-remapping-a-digit.py
class Solution:
    def minMaxDifference(self, num: int) -> int:
        s = str(num)
        maxi = s
        for i in s:
            if i != '9':
                maxi = s.replace(i, '9')
                break
        mini = s.replace(s[0],'0')
        return int(maxi) - int(mini)

File: 2704-maximum-difference-by-remapping-a-digit/2704-maximum-difference-by-remapping-a-digit.cpp
class Solution {
public:
    int minMaxDifference(int num) {
        string s = to_string(num);
        string maxi = s, mini = s;
        for (char c: s){
            if (c != '9'){
                replace(maxi.begin(), maxi.end(), c, '9');
                break;
            }
        }
        replace(mini.begin(), mini.end(), s[0], '0');
        return stoi(maxi) - stoi(mini); 
    }
};